source/scripting_v3/ScriptHookVDotNet_APIv3.csproj

```diff
@@ -26,13 +26,15 @@
     <FileVersion>3.6.0</FileVersion>
     <AssemblyVersion>3.6.0</AssemblyVersion>
     <Authors>crosire &amp; kagikn &amp; contributors</Authors>
+		<AssemblyTitle>The developer SDK for ScriptHookVDotNet v3 scripts for Grand Theft Auto V</AssemblyTitle>
+		<Product>ScriptHookVDotNet</Product>
```

Comment:
The same thing as in the v3 project file, use spaces for indentation like other lines.